packages/vscode-extension/src/exp/treatmentVariables.ts

```diff
@@ -3,9 +3,16 @@ export class TreatmentVariables {
   public static readonly EmbeddedSurvey = "embeddedsurvey";
   public static readonly CustomizeTreeview = "customizetreeview";
   public static readonly WelcomeView = "welcomeview";
+  public static readonly PreviewTreeViewCommand = "previewtreeviewcommand";
```

Comment:
"PreviewTreeView" related codes have been removed in `dev` branch, is this caused by merging?

Reply:
Yeah, it should be. Let me clean it up.